src/renderer/features/governance/components/ReferendumDetails/ProposerName.tsx

```diff
@@ -30,21 +31,26 @@ export const ProposerName = ({ referendum }: Props) => {
     fn: (loading, [proposer]) => loading && !proposer,
   });
 
-  if (!isProposerLoading && !proposer) {
-    return null;
-  }
-
-  const proposerName = proposer ? (
+  const proposerName = proposer?.parent ? (
     <AccountAddress
       addressFont="text-text-secondary"
       size={16}
       address={proposer.parent.address}
       name={proposer.parent.name || proposer.email || proposer.twitter || proposer.parent.address}
     />
+  ) : (referendum as OngoingReferendum)?.submissionDeposit?.who ? (
+    <AccountAddress
+      addressFont="text-text-secondary"
+      size={16}
+      address={(referendum as OngoingReferendum).submissionDeposit!.who}
+      name={toAddress((referendum as OngoingReferendum).submissionDeposit!.who, { chunk: 6 })}
```

Comment:
can you pass chain prop and add correct `addressPrefix` please?

Reply:
Sure, I passed it through the props since we had no chain related info in this component.